internal/pkg/parser/virtualservice_test.go

```diff
@@ -28,3 +28,27 @@ func TestParseVirtualServices(t *testing.T) {
 		}
 	}
 }
+
+func TestParseMultipleVirtualServices(t *testing.T) {
+	expectedTestCases := []*v1alpha3.VirtualService{{Spec: networkingv1alpha3.VirtualService{
+		Hosts: []string{"www.example.com", "example.com"},
+	}}}
+	testcasefiles := []string{"../../../examples/virtualservice_test.yml"}
+	configfiles := []string{"../../../examples/multidocument_virtualservice.yml"}
+	parser, err := New(testcasefiles, configfiles)
+	if err != nil {
+		t.Errorf("error getting test cases %v", err)
+	}
+	if len(parser.VirtualServices) == 0 {
+		t.Error("virtualservices is empty")
+	}
+	if len(parser.VirtualServices) < 2 {
+		t.Error("did not parse all virtualservices in file")
+	}
+
+	for _, expected := range expectedTestCases {
```

Comment:
This test could be improved if the `spec.hosts` were different in the two documents. I can do it in a separate PR.